app/scripts/services/dimStoreService.factory.js

```diff
@@ -1593,5 +1602,22 @@
       return ret;
     }
     // code above is from https://github.com/DestinyTrialsReport
+
+    function loadJSON(file) {
+      return $http.get(file)
+               .then(function(json) {
+                 return json.data.itemHash;
+               });
+    }
+
+    function getMissingSourceHashes(itemHash, missingSources) {
+      if (missingSources.$$state.value) {
```

Comment:
can you give me an example usage, ive tried changing `loadJSON` to
```
    function loadJSON(file) {
      return $http.get(file)
               .then(function(json) {
                 return json.data
                 .then(function(data) {
                   return data;
                 });
               });
    }
```
and the promise is still not resolving...